### PR TITLE
Extract reaction tests from integration-test

### PR DESCRIPTION
**Pull request by TarekkMA**
Based on #74

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -7,4 +7,5 @@ members = [
     "node",
     "runtime",
     "pallets/*",
+    "pallets/*/tests",
```
Comment (F3Joule) [inline Cargo.toml:10]: This line will probably conflict with #83

File: pallets/reactions/tests/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,62 @@
+[package]
+name = 'pallet-reactions-tests'
+version = '0.1.5'
```
Comment (F3Joule) [inline pallets/reactions/tests/Cargo.toml:3]:
```suggestion
version = '0.1.7'
```

File: pallets/reactions/tests/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+authors = ['DappForce <dappforce@pm.me>']
+edition = '2021'
+license = 'GPL-3.0-only'
+homepage = 'https://subsocial.network'
+repository = 'https://github.com/dappforce/dappforce-subsocial-node'
```
Comment (F3Joule) [inline pallets/reactions/tests/Cargo.toml:8]:
```suggestion
repository = 'https://github.com/dappforce/subsocial-parachain'
```

File: pallets/reactions/tests/Cargo.toml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+description = 'Space management pallet'
```
Comment (F3Joule) [inline pallets/reactions/tests/Cargo.toml:9]:
```suggestion
description = 'Pallet reactions tests'
```

File: pallets/reactions/tests/Cargo.toml [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+keywords = ['blockchain', 'cryptocurrency', 'social-network', 'news-feed', 'marketplace']
+categories = ['cryptography::cryptocurrencies']
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+impl-trait-for-tuples = '0.2'
+codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = ["derive"] }
+scale-info = { version = "2.1.1", default-features = false, features = ["derive"] }
```
Comment (F3Joule) [inline pallets/reactions/tests/Cargo.toml:19]:
```suggestion
scale-info = { version = "2.1.2", default-features = false, features = ["derive"] }
```

**Review by F3Joule — CHANGES_REQUESTED** [review]
Remove redundant parts of code from `tests_utils.rs` and imports from `mock.rs`, there are lots of "unused" warnings.

**Review by olehmell — APPROVED** [review]

**Review by olehmell — APPROVED** [review]

**Review by F3Joule — APPROVED** [review]

File: Cargo.toml (hunk 1 above)
Comment (F3Joule) [reply Cargo.toml:10]: Oh. I'm surprised it didn't :)

**Pull request merged**